Sources/DequeModule/Deque._Storage.swift

```diff
@@ -27,6 +27,8 @@ extension Deque {
   }
 }
 
+extension Deque._Storage: @unchecked Sendable {}
```

Comment:
Does it make sense to declare `Deque._Storage` sendable, rather than only `Deque` itself?

Reply:
There already is a Sendable conformance for `Deque`. I'm not clear why the compiler is tripping over `_Storage`'s sendability status, but setting it separately fixes the error. It should, however, depend on the `Element` type.

Reply:
What I mean is that `Deque`'s conformance ought to be `@unchecked`.

Reply:
Made `Deque.Iterator`'s conformance unchecked-Sendable; thanks for the nudge.